src/app/mainpage/components/SearchBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { SearchOutlined, InfoCircleOutlined } from "@ant-design/icons";
+import { Row, Col, Input, Select, Tooltip } from "antd";
+
+function SearchBar() {
+  return (
+    <Row style={{ width: "100%", margin: "auto" }}>
+      <Col span={18}>
+        <Input
+          placeholder="Search"
+          prefix={<SearchOutlined style={{ color: "rgba(0,0,0,.25)" }} />}
```

Comment:
you can utiilize your tokens in the components as well via useToken hook, use that here in the color style property